.gitignore

```diff
@@ -1,5 +1,4 @@
 # node files
-dist
```

Comment:
When merging to this repository it is not needed to commit dist folder as this is used via npm package and it does build its own dist when installing it. In our branch, we fetch the plugin straight from Github, and then there is a needed build step before committing to GH. 😄